### PR TITLE
ObjectLoader finetuning

### PR DESCRIPTION
**Pull request by kkananen**
- remove extra files
- filter out cameras & lights from loaded three.js scenes
- fix texture path

### CODE REVIEW

File: browser/vendor/three/three.js [hunk 1]
```diff
@@ -16037,7 +16037,10 @@ THREE.ObjectLoader.prototype = {
 
 				scope.manager.itemEnd( url );
 
-			} );
+			}, undefined,
+			function() {
```
Comment (kschzt) [inline browser/vendor/three/three.js]: please find a way to submit this back to three.js as a pull request.

File: browser/scripts/loaders/multiObjectLoader.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ function MultiObjectLoader(manager) {
 	this.objectLoader = new THREE.ObjectLoader(manager)
```
Comment (kschzt) [inline browser/scripts/loaders/multiObjectLoader.js]: doesn't this remove ObjectLoader entirely?

Comment (kkananen) [inline browser/scripts/loaders/multiObjectLoader.js]: no, ObjectLoader is included in the main three.js file. This is removing a duplicate

**Comment by kschzt** [comment]
lgtm

**Pull request merged**